pkg/internal/webhook/validator_test.go

```diff
@@ -472,10 +472,7 @@ func Test_validatorHandle(t *testing.T) {
 				WithScheme(policyapi.GlobalScheme).
 				Build()
 
-			decoder, err := admission.NewDecoder(policyapi.GlobalScheme)
-			if err != nil {
-				t.Fatal(err)
-			}
+			decoder := admission.NewDecoder(policyapi.GlobalScheme)
```

Comment:
`NewDecoder` now panics instead of returning an error https://github.com/kubernetes-sigs/controller-runtime/pull/2134/commits/927c09e54313d6476b5780b7f029f381d777c149#diff-80f5a051498d0c9b056952ccabe9e604c28b4061673003cd152d46581b25e785R35-R39